docker-setups/angular-docker.md

```diff
@@ -256,30 +258,37 @@ Run the following command to build and compose your docker image using docker-co
 ```shell
 docker-compose -f docker-compose.yml up
 ```
-### OR Alternatively build step by step
+### Build using Docker Compose
 
-#### Build Docker Image
+#### Create docker-compose.yml file
 
-Run the following command in the root directory of your project to build the Docker images. Replace `your-app-name` with a suitable value as an argument:
+Create a file named docker-compose.yml in the root directory of your project. This file will contain the configuration for your Docker services.
 
-```shell
-docker build --build-arg APP_NAME=your-app-name -t ng-docker-app:v1.0.0 -f ./Dockerfile .
-```
+[4.2 Docker Compose](#42-docker-compose)
 
-#### Create Docker Containers
+#### Build Docker images using Docker Compose
 
-Start the Docker containers using the following command:
+Instead of manually building the Docker image, you'll use Docker Compose to build it based on the configuration in docker-compose.yml. Run the following command:
 
 ```shell
-docker run -p 5000:80 -d ng-docker-app:v1.0.0
+docker-compose build
 ```
 
-#### Optional: To get the list of currently running containers
+#### Run Docker containers using Docker Compose
+Once the image is built, you can use Docker Compose to start the containers:
+```shell
+docker-compose up -d
+```
+This command will start the containers defined in the docker-compose.yml file in detached mode.
 
+#### Optional: List currently running containers:
+You can use Docker Compose to view the list of currently running containers:
 ```shell
-docker container ls
+docker-compose ps
```

Comment:
I think this new command won't work. Keep it same as earlier.
We just want to use docker-compose for building and running containers

Reply:
Reverted back.